include/verilated_vcd_sc.cpp

```diff
@@ -17,7 +17,6 @@
 //=============================================================================
 // SPDIFF_OFF
 
-#include "verilatedos.h"
```

Comment:
I'd like to keep verilatedos.h first include in every file, even if later file includes something more specific.